### PR TITLE
Development docs update

### PR DESCRIPTION
**Pull request by marians**
Attempt to improve the happa dev docs a bit

### CODE REVIEW

**Comment by marians** [comment]
Rather minor changes here. Broadening the reviewers circle...

File: README.md [hunk 1]
```diff
@@ -4,63 +4,78 @@
...
+when running happa locally:
+
+- (A) Bring up the API suite locally. See [giantswarm/api/](https://github.com/giantswarm/api/tree/master/testing) for details.
+- (B) Modify the base config in [index.html](https://github.com/giantswarm/happa/blob/master/src/index.html) to point to a test installation.
+
+### For (A) – Bring up the API suite locally
 
+This requires Docker and `docker-compose`.
 
 Start the `api` and dependencies first by going to the `api` repo and running
```
Comment (tuommaki) [inline README.md]:
```suggestion
Start the `api` and dependencies first by going to the `api` [repo](https://github.com/giantswarm/api) and running
```

File: README.md [hunk 2]
```diff
@@ -4,63 +4,78 @@
...
 make up
 ```
 
 As part of `make up`, `./fixtures.sh` will run and create the initial user and
 organization you can log in with.
 
-You should now be able to start happa's development server with:
+### For (B)
 
+Find the `apiEndpoint` setting and set it to the API endpoit of the test
```
Comment (tuommaki) [inline README.md]:
```suggestion
Find the `apiEndpoint` setting in `src/index.html` and set it to the API endpoint of the test
```

File: README.md [hunk 3]
```diff
@@ -4,63 +4,78 @@
...
+installation you want to use.
+
+If you want to test password resetting and other user account transactions,
+also adapt `passageEndpoint`.
```
Comment (tuommaki) [inline README.md]:
```suggestion
also adapt `passageEndpoint` value.
```

**Pull request merged**